gulpfile.js

```diff
@@ -40,113 +48,115 @@ var browserify = require('browserify')
 
 var liveReload = true;
 
-gulp.task('clean', function() {
-  return gulp.src(['./app/ngAnnotate', './app/dist'], { read: false })
-  .pipe(clean());
+gulp.task('clean', function () {
+    return gulp.src([paths.root + 'ngAnnotate', paths.dist], {read: false})
+        .pipe(vinylPaths(del));
```

Comment:
Another thing: For pull requests it is a good idea to have your code formatter settings match the original code style (tabs versus spaces, indentation etc.).